### DISCUSSION TITLE
Exclude data from DACE

### DISCUSSION THREAD

**Discussion by joelguerrero** [snl-dakota/dakota#27] [Q&A]
Hi,

I want to generate a dataset using DACE but I would like to be sure that in the new dataset a set of point is excluded. 

Or from another point of view, I would like to generate a new dataset but the new points shouldn't be too close to the previous dataset.

Is this possible?

joel

**Comment by jadamstephens** [comment]
I can't think of a way to do that using the DACE method. Using sampling, there are a couple of possibilities.

One is incremental LHS. You specify an initial number of samples, then a list of [refinement_samples](https://dakota.sandia.gov/sites/default/files/docs/6.17.0-release/user-html/usingdakota/reference/method-sampling-refinement_samples.html). The `refinement_samples` are placed in such a way that the "Latin" property of the design is maintained.

Another option is the [d_optimal](https://dakota.sandia.gov/sites/default/files/docs/6.17.0-release/user-html/usingdakota/reference/method-sampling-d_optimal.html) keyword. If you do a sampling study with `sample_type random` (plain Monte Carlo, not LHS) and `refinement_samples`, each addition of samples will be chosen by generating (by default) 100 candidate designs and selecting the one that maximizes the D-optimality of the total design.
